mmdet/models/roi_heads/test_mixins.py

```diff
@@ -72,11 +72,18 @@ def simple_test_bboxes(self,
             # some detector with_reg is False, bbox_pred will be None
             bbox_pred = bbox_pred.split(
                 num_proposals_per_img,
-                0) if bbox_pred is not None else [None, None]
+                0) if bbox_pred is not None else (None, ) * len(proposals)
         else:
             # some detector with_reg is False, bbox_pred will be None
-            bbox_pred = (
-                bbox_pred, ) if bbox_pred is not None else [None, None]
+            if bbox_pred is not None:
+                bucket_cls_preds, bucket_offset_preds = bbox_pred
+                bucket_cls_preds = bucket_cls_preds.split(
+                    num_proposals_per_img, 0)
+                bucket_offset_preds = bucket_offset_preds.split(
+                    num_proposals_per_img, 0)
+                bbox_pred = tuple(zip(bucket_cls_preds, bucket_offset_preds))
+            else:
+                bbox_pred = (None, ) * len(proposals)
```

Comment:
If the bbox prediction of another detector other than SABL is tuple, then we split it as `bucket_cls_preds, bucket_offset_preds = bbox_pred` may be stuck. So move the code sniper to sabl_head.py maybe a better choice.

Reply:
@yhcao6 Thanks for your suggestion, I have finished the modification now, look forward to a further review, thanks

Reply:
Thanks for your contribution, approved